### DISCUSSION TITLE
close URL tab when toggle off?

### DISCUSSION THREAD

**Discussion by mundorfd** [ttscoff/bunch-docs#216] [Q&A]
I have several bunches that include URLs to open in my default browser (Chrome). Is there a way to have those URL tabs close when toggling the bunch off?

**Comment by ttscoff** [comment]
I mean, you can have Bunch send a ⌘W to Chrome, but no, there's no way for Bunch to know what tabs to close.

**Comment by mundorfd** [comment]
Makes sense. I thought it might be able to close any URLs opened as part of the script and I was missing something. Thanks Brett!

From: Brett Terpstra ***@***.***>
Date: Tuesday, May 17, 2022 at 10:48 AM
To: ttscoff/bunch ***@***.***>
Cc: Mundorff, Deborah M ***@***.***>, Author ***@***.***>
Subject: Re: [ttscoff/bunch] close URL tab when toggle off? (Discussion #216)

[This email originated from outside of OSU. Use caution with links and attachments.]

I mean, you can have Bunch send a ⌘W to Chrome, but no, there's no way for Bunch to know what tabs to close.

**Comment by taylorjason** [comment]
Keyboard Maestro will allow you to target a Tab based on the name of the tab. You could call a KM Macro that will close the tab.
